flexmeasures/ui/templates/sensors/status.html

```diff
@@ -1,363 +1,306 @@
-                            <td colspan="6" class="text-center">
-                                No data available in table.
-                            </td>
-                        </tr>
-                    `);
-                    // Update the last call time
-                    const timeAgo = getTimeAgo(lastCallTime);
-                    $('#jobs_time_ago').html(`(${timeAgo})`);
-                    $('.jobs-time-ago').data('timestamp', lastCallTime);
-                }
-            },
-            error: function(xhr) {
-                console.error('Error fetching jobs:', xhr);
+          url: `/api/v3_0/assets/${assetId}/jobs`,
+          method: "GET",
+          success: function (res) {
+            if (res.redis_connection_err) {
+              $("#redis_connection_err").removeClass("d-none").text(res.redis_connection_err);
+            } else {
+              $("#redis_connection_err").addClass("d-none").text("");
             }
-        })   
-    }
-    
-    $(document).ready(function() {
-        $('#scheduling_forecasting_jobs').DataTable({
-            "order": [[ 0, "desc" ]],  // Default sort by the hidden UTC Timestamp column
-            "columnDefs": [
-                { 
-                    "targets": 1,  // Target the visible "Created at" column
-                    "orderData": 0  // Use data from the first column (UTC Timestamp) for sorting
-                }
-            ],
-            "searching": false,
-            "paging": false,
-            "info": false
+            const jobs = res.jobs.length ? res.jobs.map(JobRow) : [];
+            callback({ data: jobs });
+
+            const lastCallTime = Date.now();
+            $("#jobs_time_ago").html(`(${getTimeAgo(lastCallTime)})`);
+            $(".jobs-time-ago").data("timestamp", lastCallTime);
+          },
+          error: function (xhr) {
+            console.error("Error fetching jobs:", xhr);
+            callback({ data: [] });
+          },
         });
+      },
+    });
 
-        // Add a loading row to the jobs table
-        $('#scheduling_forecasting_jobs tbody').append(getLoadingRow());
-
-        // Initial fetch of jobs
-        getAssetJobs(assetId);
-        // Set up an interval to fetch jobs every 60 seconds
-        // This is to ensure that the table is updated with the latest job data
-        setInterval(function() {
-            getAssetJobs(assetId);
-        }, 60000); // Fetch jobs every 60 seconds
-
-        $("#refresh_jobs").click(function() {
-            getAssetJobs(assetId);
-        });
+    $.fn.dataTable.ext.errMode = 'none';
+
+    function getTimeAgo(timestamp) {
+      const seconds = Math.floor((Date.now() - timestamp) / 1000);
+      const intervals = [
+          { label: "year", seconds: 31536000 },
+          { label: "month", seconds: 2592000 },
+          { label: "day", seconds: 86400 },
+          { label: "hour", seconds: 3600 },
+          { label: "minute", seconds: 60 },
```

Comment:
A function with this name ("getTimeAgo") was also used before, it is in `ui/static/js/flexmeasures.js` - why did you re-define it here?

In any case, this (new) definition will not show differences in seconds (e.g. "10 seconds ago") which the old/current implementation did. And as we update every 10s, that is needed.

So you should switch back to the original. Just delete this one.